content/blog/accessing-dtap-in-pods.md

```diff
@@ -0,0 +1,264 @@
+
+```xml
+<configuration>
+  <property>
+    <name>fs.dtap.impl</name>
+    <value>com.bluedata.hadoop.bdfs.Bdfs</value>
+  </property>
+
+  <property>
+    <name>fs.AbstractFileSystem.dtap.impl</name>
+    <value>com.bluedata.hadoop.bdfs.BdAbstractFS</value>
+  </property>
+
+  <property>
+    <name>fs.dtap.impl.disable.cache</name>
+    <value>false</value>
+  </property>
+</configuration>
+```
+
+### Alternative
+
+I have prepared an example configuration file on [Github](https://github.com/helloezmeral/hpe-binary/tree/main/hadoop-dtap-config). If your Hadoop does not have a special configuration, you can simply download and replace your existing configuration file.
+
+## Test your HDFS command
+
+Here are some common commands used to interact with DataTap.
+
+```bash
+# bash: Current working directory -> $HADOOP_HOME
+
+# Check the version of the Hadoop.
+bin/hadoop version
+
+# List the files from the default TenantStorage Data Source.
+bin/hdfs dfs -ls dtap://TenantStorage/
+
+# Make new directory user in dtap://TenantStorage/.
+bin/hdfs dfs -mkdir dtap://TenantStorage/user
+
+# Move the text files helloworld.txt to "cenz" folder.
+bin/hdfs dfs -put helloworld.txt dtap://TenantStorage/cenz
+bin/hdfs dfs -put -f helloworld.txt dtap://TenantStorage/cenz # force replacement
+
+# Concatenate a file in dtap.
+bin/hdfs dfs -cat dtap://TenantStorage/cenz/helloworld.txt
+
+# Remove a file in dtap.
+bin/hdfs dfs -rm dtap://TenantStorage/cenz/helloworld.txt
+```
+
+> Tips:
+> 
+> - to get rid of the file path ```bin/```, we can add the Hadoop ```bin``` and ```sbin``` file to ```$PATH```
+> 
+> ```
+> export HADOOP_HOME=$HOME/hadoop
+> export PATH=$PATH:$HADOOP_HOME:$HADOOP_HOME/sbin:$HADOOP_HOME/bin
+> ```
+
+
+> Reference: 
+> [Hadoop File System Shell Document](https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-common/FileSystemShell.html)
+
+# Access DataTaps using pyspark
+
+## Introduction
+
+PySpark is an interface for Apache Spark in Python. Apache Spark is a unified analytics engine for big data processing, with built-in modules for streaming, SQL, machine learning and graph processing. Apache Spark can access data from HDFS and, with extension, file systems managed by DataTap. 
+
+## Install ```pyspark```
+
+There are lots of way to install Spark. The simplest way is to install pyspark package directly using ```pip install pyspark```. Run the following to install the prerequisite packages and pyspark.
+
+```bash
+# install pyspark & Java
+apt-get install python3 -y
+apt-get install python3-pip -y
+DEBIAN_FRONTEND=noninteractive apt-get install openjdk-11-jdk-headless -y
+pip install pyspark
+```
+
+There are two ways to interact with pyspark. The first one is to execute the ```pyspark``` command in bash to initiate the pyspark session. The second way is that to treat pyspark as a module which ```python``` kernel can import to. (```import pyspark```)
```

Comment:
last part of the last sentence should read "...as a module **that the** python kernel can import to."